x-pack/plugins/security_solution/public/management/cypress/test/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// For the source of these roles please consult the PR these were introduced https://github.com/elastic/kibana/pull/81866#issue-511165754
+export enum ROLES {
```

Comment:
I think we should use our `enum` here: https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_endpoint_api_int/services/roles_users.ts#L21-L31